lib/src/opentdf.ts

```diff
@@ -350,6 +357,12 @@ export class OpenTDF {
         true,
         ['sign', 'verify']
       );
+
+    this.services = CreatePlatformClient({
```

Comment:
This isn't a great API since it:

- allocates greedily - most applications won't use most of these services 
- isn't tree shakeable - if an applicaiton doesn't use one or more services, or any service at all, esbuild or other tree shaking app bundlers will include its source code, as it is still statically reachable

Maybe hold off on this for now and let the examples use `import {CreatePlatformClient} from '@opentdf/sdk/platform'` for now, or something similar?

Reply:
It's a good point that it won't be tree-shakeable. 

Without adding the services to OpenTDF it would look something like this:
```
import {createPlatformClient} from '@opentdf/sdk/platform'

const client = new OpenTDF({
      authProvider: oidcClient,
      platformUrl: '/api',
});

const platfrom = createPlatformClient({
      authProvider: oidcClient,
      platformUrl: '/api',
})
try {
  await platform.services.v1.keyAccessServerRegistry.listKeyAccessServers({});
}```

If somebody needs the platfrom and the openTDF they would need to create both. Not necessary a bad thing.


Before doing the update, let me check how much is actually being added in the bundle. I can check by verifying the size of the build of the web-app before and after

Reply:
Alright, numbers are in. Yeah, its peaty much ends up adding the size of the `dist/platform` folder.

**before**
```dist/assets/index-DQKHxfIZ.js       348.63 kB │ gzip: 110.69 kB```

**after**
```dist/assets/index-CsYy-Uz_.js       611.58 kB │ gzip: 174.03 kB```

Its about 300Kb and 60Kb gzipped.

I think for now, for the consumer to instantiate a platform client besides the OpenTDF is a good compromise.
I will also update the `createPlatformClient` into a class, so that it can be used with the same pattern as the OpenTDF class, meaning:

```
const platfrom = new PlatformClient({
      authProvider: oidcClient,
      platformUrl: '/api',
})
```

Reply:
Refactored and updated the docs

Reply:
Without some kind of test it is hard to say, but I'm pretty sure most bundlers will only include for (most of) the new code if application code accesses it via the /platform import.